### ISSUE TITLE
Expo SDK44 - Gesture Handler Issues

### ISSUE THREAD

**Issue by AhmadHoranieh** [leecade/react-native-swiper#1323]
This library completely breaks when I upgrade to the latest gesture handler version. Which is necessary to use the latest react-native-reanimated functionality. Are there plans to upgrade this library or should I move on?

**Comment by asos-gurpreetsingh** [comment]
Yep this is breaking while swiping, indexes go wrong and wrong images show up and wrong dots show up etc.

**Comment by asos-gurpreetsingh** [comment]
@AhmadHoranieh  what version works for you ?

**Comment by baruishigold** [comment]
@AhmadHoranieh @asos-gurpreetsingh 
Could you please let me know if it works with the latest Expo. I am using SDK45 in my project. 
Thank you :)

**Comment by AhmadHoranieh** [comment]
@baruishigold @asos-gurpreetsingh Sorry for the late response. Nothing currently works with this library, you would have to move on if you plan on using any of the new animation functionality.
